src/recommendations/data/loadingCoursesPlaceholders.js

```diff
@@ -0,0 +1,36 @@
+const placeholderCourse = {
+  activeCourseRun: {
+    key: 'course',
+    marketingUrl: '/',
+    type: 'Verified and Audit',
+  },
+  cardType: 'course',
+  image: {
+    src: './',
+  },
+  inProspectus: true,
+  objectID: 'skeleton',
+  owners: [{
+    key: 'skeleton',
+    logoImageUrl: './',
+    name: 'skeleton',
+  }],
+  position: 0,
+  prospectusPath: './',
+  queryID: 'skeleton',
+  recentEnrollmentCount: 0,
+  title: 'skeleton',
+  topics: [{
+    topic: 'skeleton',
+  }],
+  uuid: 'skeleton',
+};
+
+const loadingCoursesPlaceHolders = [
```

Comment:
In trending there are 3 products. is it ok if we show the skeleton for 4 products?

Reply:
yep. this is what the design says.